### PR TITLE
fix(gumby): set shell to false

### PR DESCRIPTION
**Pull request by dhasani23**
## Problem

A previous PR set shell to true for a shell command we run. This can lead to issues since special characters such as $ will be interpreted by the shell.

A smaller issue: the summary.md file was not opening automatically due to the "await" keyword.

## Solution

Set shell to false. Remove "await" keyword.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/codewhisperer/service/transformationResultsViewProvider.ts [hunk 1]
```diff
@@ -302,8 +302,8 @@ export class ProposedTransformationExplorer {
                     codeTransformApiNames: 'ExportResultArchive',
                     codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
                     codeTransformJobId: transformByQState.getJobId(),
-                    codeTransformApiErrorMessage: e?.message || errorMessage,
-                    codeTransformRequestId: e?.requestId,
+                    codeTransformApiErrorMessage: (e as Error).message ?? errorMessage,
```
Comment (nkomonen-amazon) [inline src/codewhisperer/service/transformationResultsViewProvider.ts:305]: If you put something like this right after the `catch`:

```typescript
if (!(e instanceof Error)) {
  throw e
}
```
then anything after that will have `e` typed as an `Error` and you can skip `e as Error`

**Review by nkomonen-amazon — APPROVED** [review]

File: src/codewhisperer/service/transformationResultsViewProvider.ts (hunk 1 above)
Comment (dhasani23) [reply src/codewhisperer/service/transformationResultsViewProvider.ts:305]: Makes sense, but we also use `e as Error` in many places (search here: https://github.com/aws/aws-toolkit-vscode/blob/master/src/codewhisperer/service/transformByQHandler.ts), so maybe it's better if I leave this as it is for consistency? Either that or change it everywhere, not sure which is better practice

Comment (nkomonen-amazon) [reply src/codewhisperer/service/transformationResultsViewProvider.ts:305]: We can leave it as is for now, don't want to block. But in the future if you need to use it a lot the above snippet will help.

**Pull request merged**